Ukoly/DU_7/Hangman/Hangman.py

```diff
@@ -0,0 +1,101 @@
+from random import choice
+from Extras import get_pictures
+
+list_of_words = ['praha', 'olomouc']
+
+
+def format_of_field_single(field, position, sign):
+    start = field[:position * 2]
+    end = field[(position + 1) * 2:]
+    field = '{}{}{}'.format(start, sign, end)
+    return field
+
+
+def format_of_field_many(field, position, sign):
+    while field.count(sign) != len(position):
+        for i in position:
```

Comment:
Nie je tento for cyklus v podstate len `n krát` vykonaná funkcia `format_of_field_single`? Hint. Áno je, tak prečo ju aj nepoužiť a zbaviť sa tak duplicity kódu.